lms/templates/discovery/result_item.underscore

```diff
@@ -0,0 +1,25 @@
+<article class="course" role="region" aria-label="<%= content.display_name %>">
+    <a href="/courses/<%- course %>/info">
```

Comment:
The accessible name for this link is going to be quite long, but I guess its ok.